client/src/components/_common/Form/FileInputDropZone.js

```diff
@@ -36,36 +20,33 @@ function FileInputDropZone({
   onRemoveFile,
   isSubmitted
 }) {
-  const [numberRejectedFiles, setNumberRejectedFiles] = useState(0);
+  const [rejectedFiles, setRejectedFiles] = useState([]);
 
   const { getRootProps, open, getInputProps } = useDropzone({
     noClick: true,
     maxSize,
     onDrop: accepted => {
       onSetFiles(accepted);
-      setNumberRejectedFiles(0);
     },
     onDropRejected: rejected => {
       if (onRejectedFiles) {
         onRejectedFiles(rejected);
+      } else {
+        const newRejectedFiles = rejected.filter(
+          f => rejectedFiles.filter(rf => rf.path === f.path).length === 0
+        );
+        setRejectedFiles([...rejectedFiles, ...newRejectedFiles]);
       }
-      setNumberRejectedFiles(rejected.length);
     }
   });
 
   const removeFile = fileIndex => {
     if (onRemoveFile) {
       onRemoveFile(fileIndex);
-      setNumberRejectedFiles(0);
     }
   };
 
-  const showFileList = files && files.length > 0;
-
-  if (isSubmitted && numberRejectedFiles > 0) {
-    // reset number of rejected files when files is submitted
-    setNumberRejectedFiles(0);
-  }
```

Comment:
The rejected files needs to be cleared still after submission (`setRejectedFiles([])`)